demos/linear_regression/linreg_setup.py

```diff
@@ -0,0 +1,342 @@
+    colTrunc : bool
+        Do the column pivoting and truncation or not.
+
+    Returns
+    -------
+    List
+        List of clients.
+    qrCoorinator
+        The coordinator object.
+    """
+    # load and align train data
+    g1 = pandas.read_csv("data/classificationA/train0.csv")
+    g2 = pandas.read_csv("data/classificationA/train1.csv")
+    g3 = pandas.read_csv("data/classificationA/train2.csv")
+    uid = g1.loc[:, ["uid"]]
+    uid = pandas.merge(uid, g2.loc[:, ["uid"]], on="uid", how="inner")
+    uid = pandas.merge(uid, g3.loc[:, ["uid"]], on="uid", how="inner")
+    g1 = pandas.merge(uid, g1, on="uid", how="inner")
+    g2 = pandas.merge(uid, g2, on="uid", how="inner")
+    g3 = pandas.merge(uid, g3, on="uid", how="inner")
+    label = g1.Outcome.values.astype(float)
+    label = label.reshape([label.size, 1])
+    dataset1 = {
+        "label": label, 
+        "trainFeatures": g1.loc[:, g1.columns[1:-1]].values
+    }
+    dataset2 = {
+        "label": None,
+        "trainFeatures": g2.loc[:, g2.columns[1:]].values
+    }
+    dataset3 = {
+        "label": None,
+        "trainFeatures": g3.loc[:, g3.columns[1:]].values
+    }
+    XTrain = g1.loc[:, g1.columns[1:-1]].values
+    XTrain = np.append(XTrain, g2.loc[:, g2.columns[1:]].values, axis=1)
+    XTrain = np.append(XTrain, g3.loc[:, g3.columns[1:]].values, axis=1)
+    YTrain = label
+
+    # load and align inference data
+    g1 = pandas.read_csv("data/classificationA/inference0.csv")
+    g2 = pandas.read_csv("data/classificationA/inference1.csv")
+    g3 = pandas.read_csv("data/classificationA/inference2.csv")
+    uid = g1.loc[:, ["uid"]]
+    uid = pandas.merge(uid, g2.loc[:, ["uid"]], on="uid", how="inner")
+    uid = pandas.merge(uid, g3.loc[:, ["uid"]], on="uid", how="inner")
+    g1 = pandas.merge(uid, g1, on="uid", how="inner")
+    g2 = pandas.merge(uid, g2, on="uid", how="inner")
+    g3 = pandas.merge(uid, g3, on="uid", how="inner")
+    dataset1["inferFeatures"] = g1.loc[:, g1.columns[1:]].values
+    dataset2["inferFeatures"] = g2.loc[:, g2.columns[1:]].values
+    dataset3["inferFeatures"] = g3.loc[:, g3.columns[1:]].values
+    XInfer = g1.loc[:, g1.columns[1:]].values
+    XInfer = np.append(XInfer, g2.loc[:, g2.columns[1:]].values, axis=1)
+    XInfer = np.append(XInfer, g3.loc[:, g3.columns[1:]].values, axis=1)
+
+    # set the machine info for coordinator and client
+    ip = socket.gethostbyname(socket.gethostname())
+    coordinatorInfo = MachineInfo(ip=ip, port="8890", token="%s:8890"%ip)
+    clientInfo1 = MachineInfo(ip=ip, port="8891", token="%s:8891"%ip)
+    clientInfo2 = MachineInfo(ip=ip, port="8892", token="%s:8892"%ip)
+    clientInfo3 = MachineInfo(ip=ip, port="8893", token="%s:8893"%ip)
+    
+    nFeatures = [dataset1["trainFeatures"].shape[1], dataset2["trainFeatures"].shape[1], dataset3["trainFeatures"].shape[1]]
+    clientIdWLabel = 0
+    parameter = {
```

Comment:
Same as above mentioned, it is weird to have multiple copies of configuration setup